fdbcli/fdbcli.actor.cpp

```diff
@@ -2946,6 +2946,19 @@ void LogCommand(std::string line, UID randomID, std::string errMsg) {
 	TraceEvent(SevInfo, "CLICommandLog", randomID).detail("Command", line).detail("Error", errMsg);
 }
 
+void replaceInputHyphen(int argc, char* argv[]) {
```

Comment:
We can't do this replacement to all arguments, since some of the arguments aren't going to be the flag names. For example, the ``--exec` command takes in some parameters to run in `fdbcli`, and we can't change the characters in that command.

I think we also want to default to hyphens, so the replacement should be from underscore to hyphen.